### ISSUE TITLE
[10277, 10879] Create signed version of desktop client

### ISSUE THREAD

**Issue by Andrew-Lahikainen** [DeskDirector/Issues#2]
In some environments the installer doesn't install Portal correctly.

Sometimes users must manually agree to run Portal because Windows reports it as being insecure.

**Comment by chris-ls** [comment]
Implemented for all versions of Windows clients.

**Comment by airtechsvcs** [comment]
How does one get a signed version?  When I download https://downloads.deskdirector.com/brand-name/client+name+here.exe   it's not signed.

**Comment by chris-ls** [comment]
@airtechsvcs 
When inspecting the properties of the exe/msi, there should be a "Digital Signatures" tab present.
Is this not showing up on your computer?

**Comment by airtechsvcs** [comment]
This is what I get...


https://snag.gy/HWe8yo.jpg


https://snag.gy/hkgR0C.jpg


I need to be able to push this out via an RMM tool and then launch it.  Pushing it out is one thing.  Successfully launching it.... No!



________________________________

**Comment by chris-ls** [comment]
@airtechsvcs 

The current certificate we have signed with is a standard Code Signing certificate.
Windows is going to show this warning until the application has established "Reputation" that it is safe.

The only way around this is to use an Extended Validation Certificate (EV Code signing certificate), which establishes immediate "Reputation".
However this is *a lot* more effort to get working and may not be possible.

So far there has only been 1 other client who has the same issue (smart screen) with rolling out the application. This suggests to me that there is a way to roll it out without encountering the smart screen popup.
I have created a ticket with our partner MSP who hopefully will be able to provide some direction to workaround this deployment issue.
